release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -134,6 +134,20 @@ A new `--node-selector` argument provides a way to identify which nodes you are
 
 For more information, see xref:../support/gathering-cluster-data.adoc#support-network-trace-methods_gathering-cluster-data[Network trace methods] and xref:../support/gathering-cluster-data.adoc#support-collecting-host-network-trace_gathering-cluster-data[Collecting a host network trace].
 
+[id="ocp-4-10-networking-metallb"]
+==== Enhancements to MetalLB load balancing
+
+The following enhancements to MetalLB and the MetalLB Operator are included in this release:
+
+* Support for Border Gateway Protocol (BGP) is added.
+* Support for Bidirectional Forwarding Detection (BFD) in combination with BGP is added.
+* Support for IPv6 and dual-stack networking is added.
+* Support for specifying a node selector on the `speaker` pods is added so you can control which nodes are used for load balancer IP addresses. This enhancement applies to layer 2 mode and BGP mode.
```

Comment:
Could you add rephrase the sentence to this?
Support for specifying a node selector on the `speaker` pods is added so you can control which nodes are used for **advertising** load balancer **service** IP addresses. This enhancement applies to layer 2 mode and BGP mode

Reply:
Yep, I can add those words :) New [PR](https://github.com/openshift/openshift-docs/pull/41476) for this RN where I incorporate your changes.